lib/fp.js

```diff
@@ -174,6 +174,26 @@ const restLeft = fn => (...spreadArgs) => {
   fn(args, ...namedArgs, callback);
 };
 
+// Identity function
+//   x - <any>, incoming value which will be returned
+// Returns: <any>, incoming value
+const id = x => x;
+
+// Async identity function
+//   x - <any>, incoming value which will be returned in callback
+//   callback - <Function>, callback to be called with first argument
+const asyncId = (x, callback) => {
+  callback(null, x);
```

Comment:
It isn't really async now, you should call the callback in `process.nextTick()`:
```suggestion
  process.nextTick(callback, null, x);
```